source/_integrations/smappee.markdown

```diff
@@ -45,6 +72,27 @@ smappee:
 
 Once Home Assistant restarted, go to Configuration > Integrations and select the Smappee integration. You will be redirected to a login page and be able to select the locations you would like to use within Home Assistant.
 
+#### Sensor
+Sensor entities are being added for the current active power usage,
+the always on active power, today's total consumption,
+total consumption during the current hour, total consumption during the last 5 minutes
+and the always on (slumber) consumption from today. In case of solar production entities for the active power production,
+today's total solar production
+and the solar production during the current hour are added as well.
```

Comment:
```suggestion
#### Sensor

Sensor entities are being added for the current active power usage,
the always-on active power, today's total consumption,
total consumption during the current hour, total consumption during the last 5 minutes
and the always-on (slumber) consumption from today. In case of solar production, entities for the active power production, today's total solar production
and the solar production during the current hour are added as well.
```

Additionally the heading level seems wrong.